addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -227,13 +227,13 @@ export class Chatter extends Component {
      */
     get toRecipientsText() {
         if (this.state.thread?.recipients.length === 0) {
-            return _t("No recipient");
+            return undefined;
```

Comment:
Ok if there's a single checkbox, but otherwise it should keep displaying "No recipient", or this looks off:
<img width="1276" alt="Screenshot 2024-06-07 at 15 56 32" src="https://github.com/odoo/odoo/assets/6569390/105b08a1-f586-4b53-a362-c47895129bd5">